pkgs/tools/admin/pulumi/default.nix

```diff
@@ -131,7 +131,7 @@ buildGoModule rec {
       ''
         mkdir -p $out/bin
         makeWrapper ${pulumi}/bin/pulumi $out/bin/pulumi \
-          --suffix PATH : ${lib.makeSearchPath "bin" (f pulumiPackages)}
+          --set LD_LIBRARY_PATH "${stdenv.cc.cc.lib}/lib
```

Comment:
You're missing a quote at the end of this line.

Reply:
and why remove the PATH change? isn't that needed for pulumi to actually see the packages?